airflow/api_connexion/openapi/v1.yaml

```diff
@@ -160,6 +160,14 @@ info:
 
     For details on configuring the authentication, see
     [API Authorization](https://airflow.readthedocs.io/en/latest/security/api.html).
+
+    # Update Mask
```

Comment:
Can you move it to ``### Update`` section or ``## Conventions`` (first preferred)?